.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+jobs:
+  build_android:
+    docker:
+      - image: circleci/android:api-28
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum "app/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum "app/build.gradle" }}
+          paths:
+            - ~/.gradle
```

Comment:
Just checking: Was this config file copied from somewhere (with more information on the specific steps)?
Note that Gradle does not save dependencies into this local `.gradle` directory. That's different from something like NPM/Yarn (which _does_ download dependencies locally into `node_modules` locally). In order to keep this config file short and simple, I think you can just remove the `restore_cache`/"Download Dependencies"`/`save_cache` steps, it should have little to no effect on the build time.

Reply:
It's basically the example CircleCI one. AFAIK, Gradle saves jar files to `~/.gradle` so there is a minor benefit to doing this.

Reply:
Very interesting. Thanks for replying. I wasn't aware of jar files in there. It's pretty odd that they cache the `.gradle` directory (which includes Gradle version specific stuff) but don't base it on the version of Gradle wrapper (which is defined in `gradle/wrapper/gradle-wrapper.properties`).
Were you able to measure the benefit somehow?

Reply:
I doubt the version of gradle matters as the dependent jars won't change.

I presume CircleCI think it's worth it